lib/dragonfly_libvips/processors/thumb.rb

```diff
@@ -65,10 +65,10 @@ def call(content, geometry, options = {})
         filename += "[page=#{input_options[:page]}]" if content.mime_type == "application/pdf"
 
         thumbnail_options = thumbnail_options.transform_keys { |k| k.to_sym } # symbolize
-        thumb = ::Vips::Image.thumbnail(filename, dimensions.width.ceil, DragonflyLibvips.symbolize_keys(thumbnail_options))
+        thumb = ::Vips::Image.thumbnail(filename, dimensions.width.ceil, **DragonflyLibvips.symbolize_keys(thumbnail_options))
```

Comment:
_maybe?_ definition of thumnbail - https://github.com/libvips/ruby-vips/blob/88e737c378c1e53e162b48527a94eebe0221e1dd/lib/vips/methods.rb#L2129-L2144